<launch>

  <node pkg="base_controller" type="sonar_safe_node_1" name="sonar_safe_node_1">
    <param name="safe_dist" value="0.251" type="double"/>
    <remap from="yocs_cmd_vel_mux/input/safe" to="yocs_cmd_vel_mux/input/remote"/>
  </node>

</launch>
